File: src/test/scala/com/kjetland/jackson/jsonSchema/testDataScala/ManyDates.scala
package com.kjetland.jackson.jsonSchema.testDataScala

import java.time.{LocalDate, LocalDateTime, OffsetDateTime}

case class ManyDates
(
  javaLocalDateTime:LocalDateTime,
  javaOffsetDateTime:OffsetDateTime,
  javaLocalDate:LocalDate,
  jodaLocalDate:org.joda.time.LocalDate
)

File: src/test/java/com/kjetland/jackson/jsonSchema/testData/ManyDates.java
package com.kjetland.jackson.jsonSchema.testData;

import java.time.LocalDate;
import java.time.LocalDateTime;
import java.time.OffsetDateTime;
import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.ToString;
import lombok.experimental.FieldDefaults;
import lombok.extern.jackson.Jacksonized;

/**
 *
 * @author alex
 */
@AllArgsConstructor
@Jacksonized @Builder
@FieldDefaults(level = AccessLevel.PUBLIC, makeFinal = true)
@ToString @EqualsAndHashCode
public class ManyDates {
    LocalDateTime javaLocalDateTime;
    OffsetDateTime javaOffsetDateTime;
    LocalDate javaLocalDate;
    org.joda.time.LocalDate jodaLocalDate;
}
